.circleci/config.yml

```diff
@@ -8,13 +8,9 @@
 version: 2.1
 orbs:
   # use Cypress orb from CircleCI registry
-  cypress: cypress-io/cypress@2.2.0
+  cypress: cypress-io/cypress@3
 
 executors:
```

Comment:
What does removing the `Mac` distinction mean for this? I don't know if the naming, etc was tracked for anything in the CircleCI dashboard.

Reply:
I am essentially telling it to use our executor instead of the `mac` machine wired up here

Reply:
I get that part. I just mean I wasn't sure what ramifications that has. Like, was that mac distinction meaningful for anything...maybe in the reporting in the dashboard or something?